CHANGELOG.rst

```diff
@@ -4,6 +4,9 @@ Change Log
 unreleased
 ~~~~~~~~~~
 
+* extend Y010 to cover what was previously included in Y090 (disallow
+  assignments in ``__init__`` methods) and Y091 (disallow ``raise``
+  statements)
```

Comment:
It would be good to emphasize that the check are no longer disabled by default. It can be guessed, if you also look at the README, but it would be good to just say it here instead of assuming the reader remembers that `Y09x` errors are special.